CONTRIBUTING.md

```diff
@@ -1,91 +1,69 @@
-* can be publicly visible (avoid confidential info)
-* Generally applicable (not specific to internal processes)
-* Short code snippets or links to OSS repos (no large code assets)
-
-## How to contribute
+> NOTE: Please discuss with one of the [maintainers](#maintainers) before adding a new top level section. Try to find a suitable home in an existing section if possible.
 
-When you have an idea for a contribution do one of two things:
+We welcome maintenance of the repository and contributions that:
 
-* Contact one of the champions in the appropriate fundamental area and discuss your idea with them (check the reviewer groups [here](https://github.com/microsoft/code-with-engineering-playbook/blob/main/.github/reviewers.yml) to find a champion in the appropriate fundamental area)
-* Look in the opened issues, if you cannot find a similar one then open an issue following the issue templates
-  * **Note:** Adding new labels for an issue are only available for the project's Contributors group. Proposals for new labels are welcomed.
+- Makes the content more readable
+- Makes the content more discoverable
+- Fixes content that has gone out-of-date
 
-Once you have discussed your contribution:
+or anything that improves the quality of the playbook
 
-* Add your changes to a new branch following [this branch naming convention](#branch-naming-convention)
-* Open a PR and reviewers should be automatically assigned based on the location of your files changed. You need 2 approvals to merge your PR.
+## How to contribute
 
-## Git guidance
+Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have
+the right to, and actually do, grant us the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
 
-Consistent with the practices suggested in this playbook, please follow the
-specifics regarding git as described in this section.
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label,
+comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-### Branch naming convention
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com] (mailto:opencode@microsoft.com) with any additional questions or comments.
 
-In this repo, we use this branch naming approach:
+### Permissions & Contributions
 
-```plaintext
-<user alias>/<work item ID>_<title>
-```
+There are two ways in which you can help update the content:
 
-### Linting
+- **One-off:** \
+If you are not a regular contributor to the project, but you would like to contribute some changes, the best way to do it is to follow these [steps](#how-to-contribute)
```

Comment:
This is a little confusing as this is forwarding to the same section

Reply:
fixed